### PR TITLE
Add SELinux details to diagnostics bundle

### PR DESCRIPTION
**Pull request by janisz**
## High-level description

Simplify debugging by adding SELinux information into diagnostics bundle

## Corresponding DC/OS tickets (obligatory)

These DC/OS JIRA ticket(s) must be updated (ideally closed) in the moment this PR lands:

  - [DCOS_OSS-4123](https://jira.mesosphere.com/browse/DCOS_OSS-4123) Get SELinux details from dcos-diagnostics bundle

## Checklist for all PRs

  - [x] Added a comprehensible changelog entry to `CHANGES.md` or explain why this is not a user-facing change:
  - [x] Included a test which will fail if code is reverted but test is not. If there is no test please explain here:
  - [x] Read the [DC/OS contributing guidelines](https://github.com/dcos/dcos/blob/master/contributing.md)
  - [x] Followed relevant code rules [Rules for Packages and Systemd](https://github.com/dcos/dcos/tree/master/docs)

### CODE REVIEW

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@mesosphere-mergebot label Ready For Review

**Comment by jgehrcke** [comment]
Wow, many CI failures here. I only looked at the first one so far and found a new instability for which I created https://jira.mesosphere.com/browse/DCOS_OSS-4130.

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/aws/cloudformation/simple https://jira.mesosphere.com/browse/DCOS_OSS-4130

**Comment by orsenthil** [comment]
Hi @janisz and @jgehrcke  - This https://jira.mesosphere.com/browse/DCOS_OSS-4130 should actually block this PR. We shouldn't override this and introduce a new flake until we can have deterministic test for collecting and verify the log bundles.

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@orsenthil It's fixed now. The problem was in a missing comma (`,`) in a configuration. The problem is now getting fixed in dcos-diagnostics https://github.com/dcos/dcos-diagnostics/pull/56 it will fail to start when the configuration is invalid.

**Comment by janisz** [comment]
@orsenthil Can you approve it now?

**Comment by orsenthil** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@orsenthil @adamtheturtle Can you take a look?

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by adamtheturtle** [comment]
@mesosphere-mergebot bump-ee

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py [hunk 1]
```diff
@@ -418,22 +418,23 @@ def _download_bundle_from_master(dcos_api_session, master_index, bundle):
...
                              'var/lib/dcos/exhibitor/conf/zoo.cfg.gz', '5050-quota.json',
                              'binsh_-c_cat proc`systemctl show dcos-mesos-master.service -p MainPID'
-                             '| cut -d\'=\' -f2`environ-10.output.gz', '5050-overlay-master_state.json.gz'
+                             '| cut -d\'=\' -f2`environ-11.output.gz', '5050-overlay-master_state.json.gz'
                              ] + expected_common_files
 
     expected_agent_common_files = ['5051-containers.json', '5051-overlay-agent_overlay.json',
                                    'binsh_-c_cat proc`systemctl show dcos-mesos-master.service -p MainPID'
-                                   '| cut -d\'=\' -f2`environ-11.output.gz']
+                                   '| cut -d\'=\' -f2`environ-12.output.gz']
```
Comment (branden) [inline packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: For my own curiosity: Why do the numbers in these filenames need to be incremented?

**Comment by branden** [comment]
@mesosphere-mergebot sync

**Comment by branden** [comment]
Looks like this PR has a merge conflict in `CHANGES.md` that needs to be resolved, and according to @mesosphere-mergebot it needs approval from @orsenthil.

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py (hunk 1 above)
Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: I don't know why we need them but all command are numbered. I asked if we can remove it but got no answe

**Comment by janisz** [comment]
@mesosphere-mergebot bump-ee

**Comment by janisz** [comment]
@dcos/admins Please merge this PR since it's already changed in EE version

**Comment by jgehrcke** [comment]
Chatted with @janisz out of band -- waiting for CI

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status teamcity/dcos/test/aws/onprem/static https://jira.mesosphere.com/browse/DCOS_OSS-4486

**Comment by janisz** [comment]
@mesosphere-mergebot sync

**Comment by jgehrcke** [comment]
@mesosphere-mergebot override-status mergebot/review/approved/min_2 https://jira.mesosphere.com/browse/DCOS_OSS-2324

File: packages/dcos-integration-test/extra/test_dcos_diagnostics.py (hunk 1 above)
Comment (MrMarvin) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: Just stumbled about this today, here is what @klueska and I found: The number is an index to the command from gen/dcos-config.yaml. By adding the `sestatus` command, the index shifted. We agreed that using an index here is either overly sensitive as it is very likely to 'detect' changes without explicitly adjusting the test case, or very smart, depending on how to look at it.

Comment (janisz) [reply packages/dcos-integration-test/extra/test_dcos_diagnostics.py]: https://jira.mesosphere.com/browse/DCOS_OSS-4155

**Pull request merged**